### PR TITLE
Update OG, add og image, update favicon image

### PR DESCRIPTION
**Pull request by neilmispelaar**
### TLDR

The project was using the default vue favicon and didn't have any Open graph data. This PR attempts to address both of those issues. In addition, the description metadata was set to `TODO` so every time someone shared the link the 'TODO' text showed up which was awkward sometimes. 

### What's in this

- Updated favicon image
- Thomas' cool open graph image
- Markup to specify a favicon icon
- Markup to specify the open graph data
- Updated metadata to remove the 'todo' text

### Testing

- When the deploy preview goes live (because you can't test OG on localhost - I mean you can, but it's not an accurate test) use a site like [https://www.opengraph.xyz/](https://www.opengraph.xyz/) to test the open graph data is set properly
- You can also share the link to the deploy preview in something like slack to see what the link preview looks like 
- To test the favicon load the deploy preview in a browser and look in the tab to see if it's a Canada Maple Leaf and not the green 'v'

### CODE REVIEW

**Comment by neilmispelaar** [comment]
Stupid image isn't bloody loading: 

Screen Shot 2021-06-25 at 9 19 12 AM

**Comment by neilmispelaar** [comment]
Works in IOS: 

Screen Shot 2021-06-25 at 9 44 30 AM

**Comment by neilmispelaar** [comment]
Works in Slack without the Full URL for the OG image:

Screen Shot 2021-06-25 at 2 40 50 PM

**Comment by neilmispelaar** [comment]
### Recommendation

I'm going to recommend we merge this in. 

### Rationale

This PR fixes the "TODO" metadata description and Open Graph does work on some systems (like Slack, iOS, MS Teams - see screenshots above). 

### Remaining issue

The Open Graph specification doesn't say you need a full absolute URL for the OG image property (although the sample code does include a full url - figure that one out). 

### Reference

[https://ogp.me/](https://ogp.me/)

**Comment by neilmispelaar** [comment]
Please feel free to review at your convenience. 

**Possible issue:** 

- Favicon duplication / overwrite

**Review by nibivi77 — APPROVED** [review]
Thank you Neil. i don't think there should be a merge conflict with the favicon.
